packages/application-generic/src/services/bull-mq/bull-mq.service.ts

```diff
@@ -174,6 +201,8 @@ export class BullMqService {
       await this._worker.close();
     }
 
+    await this.inMemoryProviderService.shutdown();
```

Comment:
This ensures clean exits in testing suites such Jest as open handles could be left.